### PR TITLE
[4.2] typos/grammar

### PR DESCRIPTION
**Pull request by brianteeman**
I was testing a script using the joomla repo and found these typos/grammar errors.

The basic english language rule is that if you want to write "a something" and the "something" begins with a vowel then it changes to "an"

This also applies even if the "something" _sounds_ like it begins with a vowel.

As these are all corrections in comments testing is just by review

### CODE REVIEW

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 52d7ce02793977a1ce5fb74b6903a1455c418447<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39432">issues.joomla.org/tracker/joomla-cms/39432</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 52d7ce02793977a1ce5fb74b6903a1455c418447<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39432">issues.joomla.org/tracker/joomla-cms/39432</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39432">issues.joomla.org/tracker/joomla-cms/39432</a>.</sub>

File: build/build-modules-js/utils/hashfromfile.es6.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { createHash } = require('crypto');
 const { createReadStream } = require('fs');
 
 /**
- * Get an SHA1 hash for a given file
+ * Get a SHA1 hash for a given file
```
Comment (richard67) [inline build/build-modules-js/utils/hashfromfile.es6.js:5]: Doesn’t „es“ (spoken „s“ in abbreviation „SHA“) sound like starting with a vowel?

**Comment by roland-d** [comment]
Thank you

**Comment by richard67** [comment]
@roland-d I know it’s only grammar in comments, but have you seen my review comment?

**Comment by brianteeman** [comment]
> @roland-d I know it’s only grammar in comments, but have you seen my review comment?

??

**Comment by roland-d** [comment]
@richard67 I only see that you merged `4.2-dev` branch into here. There is nothing else.

File: build/build-modules-js/utils/hashfromfile.es6.js (hunk 1 above)
Comment (richard67) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: As far as I know it‘s pronounced „es - age - a“ and so shoul be „an“

**Comment by richard67** [comment]
Ah i see the review was pending.

File: build/build-modules-js/utils/hashfromfile.es6.js (hunk 1 above)
Comment (brianteeman) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: I always pronounce it as sh -ah - one

**Comment by richard67** [comment]
Can you see it now? Review was pending still, my mistake.

**Comment by roland-d** [comment]
Now it is visible.

File: build/build-modules-js/utils/hashfromfile.es6.js (hunk 1 above)
Comment (brianteeman) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: Listen here
https://www.youtube.com/watch?v=DMtFhACPnTY&ab_channel=Computerphile
https://www.youtube.com/watch?v=kmHojGMUn0Q&ab_channel=FullstackAcademy

Comment (brianteeman) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: I hope that answers your comment

Comment (richard67) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: I see … similar thing as with SQL … I call it S.Q.L. but mainly it’s called „sequel“ … didn’t know it’s like that with SHA1, too. Learned something new now. Thanks.

Comment (brianteeman) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: es-que-el and sequel is a bit harder as both pronunciations seem to be common use.

Strangely I always say M-eye-es-que-el and es-que-el BUT sequelserver

Comment (richard67) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: We should change the project language to German as English seems to be a bit weird language ;-)

Comment (brianteeman) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: Would that be formal or informal german?

Comment (richard67) [reply build/build-modules-js/utils/hashfromfile.es6.js:5]: Hmm … let’s stay with English :-)

**Pull request merged**